### DISCUSSION TITLE
Why is winget supplying the portable version of VSCodium?

### DISCUSSION THREAD

**Discussion by mariobrostech** [microsoft/winget-pkgs#25751] [Q&A] [answered]
Hi,

Out of curiosity, I was wondering why winget fetches the "user"/portable version of VSCodium as opposed to the full desktop installer. Could anyone clarify why this might be, given that most other applications available on winget are full installers?

For reference, here are the links to the latest portable and full installers for VSCodium. winget seems to be using the former.
[Portable installer](https://github.com/VSCodium/vscodium/releases/download/1.59.1/VSCodiumUserSetup-x64-1.59.1.exe)
[Full installer](https://github.com/VSCodium/vscodium/releases/download/1.59.1/VSCodiumSetup-x64-1.59.1.exe)

If this is the intended behavior, is there any reason why the desktop installer isn't available as an option?

Thanks in advance!

**Comment by ImJoakim** [comment, marked as answer]
Heya @mariobrostech, the manifest currently contains both the `User` installer and the `Machine` installer. You can choose which installer to install by doing the following:

`winget install vscodium --scope machine`
`winget install vscodium --scope user`

It defaults to the user installer.

**Reply by mariobrostech** [reply]
I didn't know that was possible; thanks for the clarification! Is there any particular reason why it defaults to the `User` installer? I'd imagine that the `Machine` installer is the one most people want, especially since most winget packages are full installers by default.

**Reply by ImJoakim** [reply]
I don't know the reasoning off the top of my head. I do know that you are able to choose the default behavior via settings. You can read more about that [here](https://docs.microsoft.com/en-us/windows/package-manager/winget/settings).

**Reply by mariobrostech** [reply]
Thanks for the tips! I'm going to leave this discussion open so someone can comment if they know more about this behavior in particular.

**Reply by OfficialEsco** [reply]
as this is a VSCode fork AFAIK the preferred installation type is User, unless VSCodium prefer the Machine Wide installation.

WinGet probably defaults to User so the installation is as silent as possible, Microsoft have also started going in the User installation direction for Private use and Machine Wide for Company's.

Quite a lot of installers have also started to use/recommend user installation (Appx, MSIX, NSIS/Nullsoft and the latest InstallShield(?) installer)

Most users will not care or notice which is why they default to User, if you've installed the software Machine Wide before `winget upgrade` will respect that choice.

You'll probably get a more concrete answer from @denelon

**Reply by denelon** [reply]
@mariobrostech, @OfficialEsco is correct. We're defaulting to user as the standard mode. As @ImJoakim stated, we support both command line and settings to specify any preferences / requirements one has. Generally speaking, the user is default so as not to require UAC or elevation where possible.

**Reply by mariobrostech** [reply]
Ok, sounds good, thanks for the help! I'm closing this now since the solution has already been posted for those who want to change the installation scope.
